Makefile

```diff
@@ -898,58 +898,8 @@ print/env:
 
 # buildbox-grpc generates GRPC stubs
 .PHONY: buildbox-grpc
```

Comment:
Maybe now is a good time to fix this poorly named target?

I always found it odd that

- `make grpc` runs inside the buildbox container
- `make buildbox-grpc` runs directly on the host, not in the buildbox

Reply:
Any suggestions? Maybe `make grpc/internal`?

Reply:
I think keeping `make grpc` as the main command is a good idea, too much impact on our devs otherwise. I'm happy to rename the other target, but I'm open to suggestions - at this point I understand that yall don't like the target name, but idk what you are looking for.

Reply:
Renamed to `make grpc/host` and `make grpc-teleterm/host`.